src/redux/LeftSidebar/slice.ts

```diff
@@ -13,14 +14,40 @@ interface LeftSidebarState {
 
 const initialState: LeftSidebarState = {
   buttons: [
-    { id: 'dashboard', isDotOn: false, isSelected: true },
-    { id: 'speed', isDotOn: false, isSelected: false },
-    { id: 'chart', isDotOn: false, isSelected: false, isDisabled: true },
-    { id: 'heart', isDotOn: false, isSelected: false },
-    { id: 'codeBlock', isDotOn: false, isSelected: false },
-    { id: 'communities', isDotOn: false, isSelected: false },
-    { id: 'activityCenter', isDotOn: true, isSelected: false },
-    { id: 'settings', isDotOn: false, isSelected: false },
+    { id: 'dashboard', isDotOn: false, isSelected: true, path: '/dashboard' },
+    {
+      id: 'speed',
+      isDotOn: false,
+      isSelected: false,
+      path: '/validator-management',
+    },
+    {
+      id: 'chart',
+      isDotOn: false,
+      isSelected: false,
+      isDisabled: true,
+      path: '/logs',
+    },
+    {
+      id: 'heart',
+      isDotOn: false,
+      isSelected: false,
+      path: '/device-health-check',
+    },
+    { id: 'codeBlock', isDotOn: false, isSelected: false, path: '/dashboard' },
+    {
+      id: 'communities',
+      isDotOn: false,
+      isSelected: false,
+      path: '/dashboard',
+    },
+    {
+      id: 'activityCenter',
+      isDotOn: true,
+      isSelected: false,
+      path: '/dashboard',
+    },
+    { id: 'settings', isDotOn: false, isSelected: false, path: '/dashboard' },
```

Comment:
Too many paths lead to `/dashboard`. This seems odd
Also:
- `codeBlock` should lead to `/logs` ( according to [Figma](https://www.figma.com/file/kUO8PyQCo89SyvCn3pFmNS/Nodes-Nimbus---New-Design-System?type=design&node-id=101-208824&mode=design&t=fuusIOAz0mtw4z37-0) )
- `settings` should lead to `/validator-management`, not `speed`( according to [Figma](https://www.figma.com/file/kUO8PyQCo89SyvCn3pFmNS/Nodes-Nimbus---New-Design-System?type=design&node-id=101-208821&mode=design&t=9wiXB8sOvXtTTChC-0) )
- if there are buttons that lead to pages that are not implemented, better disable them.